eppo_core/src/eval/eval_bandits.rs

```diff
@@ -265,8 +265,11 @@ impl BanditModelData {
                     // In the case of multiple actions getting the same best score, we need to break
                     // the tie deterministically.
                     //
-                    // Compare action names next
-                    Ord::cmp(a.0, b.0)
+                    // Compare action names next.
+                    //
+                    // We're reversing the comparison, so that before-ordered name is considered
+                    // higher and wins the best score.
+                    Ord::cmp(a.0, b.0).reverse()
```

Comment:
We already compared by name but picking the name later in the order

Reply:
Ah do we need the `reverse()` because of the surrounding (`max_by()`)?

Reply:
Yes. `max_by()` selects the highest value, so with `"a" < "b"` it was selecting the "b"